Makefile

```
@@ -254,7 +254,7 @@ e2e-coverage:
 .PHONY: kind-load
 kind-load: $(KIND) #EXHELP Loads the currently constructed images into the KIND cluster.
 	$(CONTAINER_RUNTIME) save $(IMG) | $(KIND) load image-archive /dev/stdin --name $(KIND_CLUSTER_NAME)
-	IMAGE_REPO=$(CATALOG_IMAGE_REPO) KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) $(MAKE) -C catalogd kind-load
+	IMAGE_REPO=$(CATALOG_IMAGE_REPO) KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) $(MAKE) -f Makefile.catalogd kind-load
```

```suggestion
	$(CONTAINER_RUNTIME) save $(CATALOG_IMAGE_REPO) | $(KIND) load image-archive /dev/stdin --name $(KIND_CLUSTER_NAME)
```
We no longer need to call kind-load from the Makefile inside catalogd.
Both make run and the tests run from the root file now.

Therefore, we can follow the same approach as we did for the operator-controller image and remove the related targets from the Makefile in catalogd.

The goal is to remove the Makefile under the catalogd directory entirely.

---

That should be resolved. That was just a remnnant that needed to be cleaned